firmware/library/L1_Drivers/gpio.hpp

```diff
@@ -0,0 +1,17 @@
+#pragma once
+
+#include <stdint.h>
```

Comment:
This isn't used, so it should be removed.

Reply:
Also, standard states that you must use the C++ include of stdint.h which is `<cstdint>`

Reply:
Just to be clear, #pragma once is left in for this commit, but #include <stdint.h> is left out since it is not being used at the moment, right?

Reply:
Also when i start to use cstdint, do I use the scope resolution operator for uint8_t?

Reply:
Nope, cstdint is the same as stdint.h. No need for the scope operator.

Reply:
You need #pragma once, to be your include guard. You don't use anything from stdint here so remove that include.